source/requirements.txt

```diff
@@ -0,0 +1,2 @@
+boto3
```

Comment:
Let's add specific version for boto3 and Click that were used. So we can freeze the version for repeatability.